### PR TITLE
Feature/ju video player/add documentation

### PR DESCRIPTION
**Pull request by AdriOli134**
Add documentation for all the classes

### CODE REVIEW

File: app/src/main/java/ju/video/player/view/commons/RoundedBorder.java [hunk 1]
```diff
@@ -19,13 +19,22 @@
...
+ * @version 1.0
+ */
 class RoundedBorder extends BasicButtonUI {      
       
     public RoundedBorder() {
         super();
     }
       
+    /**
+     * Set the border of the button
```
Comment (MariaAT18) [inline app/src/main/java/ju/video/player/view/commons/RoundedBorder.java:35]: Add space between the Text and param.

File: app/src/main/java/ju/video/player/view/commons/TextField.java [hunk 2]
```diff
@@ -28,6 +34,10 @@ public TextField(String name, MaskFormatter mask) {
         initPanel(name);
     }
 
+    /**
+     * Initialize the JText
+     * @param name
```
Comment (MariaAT18) [inline app/src/main/java/ju/video/player/view/commons/TextField.java:40]: Add space between the Text and param.

File: app/src/main/java/ju/video/player/view/commons/TextField.java [hunk 3]
```diff
@@ -36,6 +46,10 @@ public void initPanel(String name) {
         setBackground(UIColor.PRIMARY_BACKGROUNG_COLOR);
     }
 
+    /**
+     * Set the Title that appears in the border
+     * @param title
```
Comment (MariaAT18) [inline app/src/main/java/ju/video/player/view/commons/TextField.java:53]: Add space between the Text and param.

File: app/src/main/java/ju/video/player/view/videoplayer/NameVideoText.java [hunk 4]
```diff
@@ -39,6 +39,10 @@ public NameVideoText(MediaPlayerFrame mediaPlayerFrame, PlayListFrame playListFr
         setBackground(UIColor.PRIMARY_BACKGROUNG_COLOR);
     }
 
+    /**
+     * For change the text of the media name field
```
Comment (MariaAT18) [inline app/src/main/java/ju/video/player/view/videoplayer/NameVideoText.java:43]: Add space between the Text and param.

File: app/src/main/java/ju/video/player/model/ListValidVideos.java [hunk 5]
```diff
@@ -106,17 +116,36 @@ public void setEndDate(LocalDate endDate) {
...
+     */
     public void restoreFilters(FiltersPanel filtersPanel) {
+        restoreUIFilters(filtersPanel);
+        restoreFiltersValue();
+        applyFilters();
+    }
+    
+    /**
+     * Restore the filters UI components to their initial state
+     * @param filtersPanel
```
Comment (MariaAT18) [inline app/src/main/java/ju/video/player/model/ListValidVideos.java:135]: Add space between the Text and param.

File: app/src/main/java/ju/video/player/model/ListValidVideos.java [hunk 6]
```diff
@@ -106,17 +116,36 @@ public void setEndDate(LocalDate endDate) {
         this.endDate = endDate;
     }
 
+    /**
+     * Restore all the filters element to the initial state
+     * @param filtersPanel
```
Comment (MariaAT18) [inline app/src/main/java/ju/video/player/model/ListValidVideos.java:124]: Add space between the Text and param.

File: app/src/main/java/ju/video/player/model/Format.java [hunk 7]
```diff
@@ -49,10 +49,18 @@ private Format(String format) {
...
+    /**
+     * @return format
+     */
     public String getFormat() {
         return format;
     }
 
+    /**
+     * Verify if the parameter is equeals to some Format in the enum
+     * @param formatt
```
Comment (MariaAT18) [inline app/src/main/java/ju/video/player/model/Format.java:62]: Add space between the Text and param.

File: app/src/main/java/ju/video/player/controller/componentscontrollers/ControlButtonsController.java [hunk 8]
```diff
@@ -25,6 +40,9 @@ public ControlButtonsController(EmbeddedMediaListPlayerComponent mediaComponent,
         isFullScrean = false;
     }
 
+    /**
+     * handle the PlayButton behavior and actions
```
Comment (MariaAT18) [inline app/src/main/java/ju/video/player/controller/componentscontrollers/ControlButtonsController.java]: h should be Capital Letter

**Review by MariaAT18 — CHANGES_REQUESTED** [review]

**Review by junior0123 — APPROVED** [review]
Good Job

**Review by MariaAT18 — APPROVED** [review]

**Pull request merged**